tests/run/return.pyx

```diff
@@ -14,6 +14,9 @@ cdef int h(a):
     i = a
     return i
 
+cdef const int p():
```

Comment:
Is there a test for getting const as an argument and returning it? If not it would be nice to add it
As it seems this case is implemented but not tested

Reply:
@0dminnimda I decided that I didn't think this was necessary to merge the PR. But I'd happily add the test if someone submits it later